resources/shaders/pp_dof_composition.hlsl

```diff
@@ -46,37 +46,42 @@ void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
 {
 	float2 screen_size = float2(0.f, 0.f);
 	output.GetDimensions(screen_size.x, screen_size.y);
+	screen_size -= 1.0f;
 
 	float2 screen_coord = int2(dispatch_thread_id.x, dispatch_thread_id.y);
 
+
 	float2 texel_size = 1.0 / screen_size;
-	float2 uv = (screen_coord + 0.5f) / screen_size;
-	float2 uvc = (screen_coord) / screen_size;
+	float2 uv = (screen_coord) / (screen_size);
 
-	float coc = GetDownSampledCoC(uvc, texel_size);//coc_buffer.SampleLevel(s0, uvc, 0);
+	//float coc = GetDownSampledCoC(uv, texel_size);
```

Comment:
Either get rid of commented code or add a comment that tells us why you keep that commented code there.